cocos/base/ZipUtils.cpp

```diff
@@ -23,6 +23,9 @@
  THE SOFTWARE.
  ****************************************************************************/
 
+// FIXME: hack, this needs to be before minizip/unzip...
+#include <string> 
```

Comment:
Why do we need to be touching header files in ZipUtils? I don't understand why this PR would need to introduce this change.

Reply:
Sorry i'm going to close this pull request...  cocos2d::RefPtr seems a sufficient wrapper class to providing similar functionality.